crates/ipfix-code-generator/src/generator.rs

```diff
@@ -284,7 +287,13 @@ pub(crate) fn generate_ie_status() -> String {
 
 /// Use at the beginning of `ie_generated` for defining custom types
 pub(crate) fn generate_common_types() -> String {
-    "pub type MacAddress = [u8; 6];\n\n".to_string()
+    //TEMP, and in the future move it to another function generate_imports
+    let mut ret = String::new();
+    ret.push_str("use netgauze_iana::tcp::*;\n\n");
```

Comment:
let's avoid the import here, just in case in the future it might create accidental conflicts. Rather use explicit paths and replace `TCPHeaderFlags` with `netgauze_iana::tcp::TCPHeaderFlags`